### ISSUE TITLE
Add support for PolyLoss (ICLR 2022)

### ISSUE THREAD

**Issue by acforvs** [deepmind/optax#466]
Hi, thanks for maintaining the library!
What are your thoughts on adding support for PolyLoss (see https://arxiv.org/pdf/2204.12511.pdf, page 10)?

Though it can be implemented externally based on the existing `softmax_cross_entropy`, I would still suggest to add it to the library alongside with some tests

I would also like to volunteer to work on that, if the change seems reasonable :)

**Comment by mkunesch** [comment]
Hi! I took a quick look at the paper and I think this would be great to have as a separate loss function in optax - thanks a lot!

Feel free to write here if you'd like to discuss the design or if you have any questions!

**Comment by acforvs** [comment]
Thanks! I've just opened a pull request with Poly-1 loss implementation, please let me know what you think
